### ISSUE TITLE
Subtract xp from previous level

### ISSUE THREAD

**Issue by Mias15** [krishsharma0413/DiscordLevelingCard#6]
Would it be possible to add the option to subtract the xp from the previous level? It would make it possible to indicate the total xp at the top but only the progress of the current level would be shown at the xp bar.

**Comment by krishsharma0413** [comment]
Please explain in again with an example. I wasn't able to understand

**Comment by Mias15** [comment]
correct; the finished code is already available in the open pull request

**Comment by krishsharma0413** [comment]
alright, I will review and merge accordingly.
